diagnostic.md

```diff
@@ -48,11 +50,11 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    {{movie.id}}??
```

Comment:
The Route model can take a params parameter. This gives us access to all route params. Thus, we'd get the movie_id of 123 in the model with params.movie_id